src/Kovan-DssSpell.sol

```diff
@@ -36,138 +27,46 @@ contract SpellAction {
     //
     // The contracts in this list should correspond to MCD core contracts, verify
     //  against the current release list at:
-    //     https://changelog.makerdao.com/releases/kovan/1.1.4/contracts.json
+    //     https://changelog.makerdao.com/releases/kovan/active/contracts.json
```

Comment:
Do we even need this now that all addresses are being pulled from the on-chain changelog?

Reply:
Brian preferred to keep it: https://github.com/makerdao/spells-kovan/pull/34#discussion_r528986005